src/brevitas/nn/quant_layer.py

```diff
@@ -357,12 +368,15 @@ def forward_impl(self, inp: Union[Tensor, QuantTensor]) -> Union[Tensor, QuantTe
             elif quant_input.zero_point is not None and output_zero_point is None:
```

Comment:
quant_input may be normal Tensor

Reply:
It can't be normal tensor because I check before for this condition and raise an error if it happens